### DISCUSSION TITLE
Students Developer Packed

### DISCUSSION THREAD

**Discussion by Arvindcx** [community/community#101643] [GitHub Education]
Hi everyone,

so the thing is I have been approved and it is saying that:
![image](https://github.com/community/community/assets/85410025/1437c434-68c7-4bf6-ac71-79de7c4ff86a)
(I couldn't find any other discussion so this can maybe also help others)
I am just curious how long it takes till I get the benefits. Any answer is appreciated!

[ Edit ] Today I Got it 😍

**Comment by fury-05** [comment]
They are activated instantly visit this link to check you have received your benefits : https://education.github.com/pack/offers

**Reply by Arvindcx** [reply]
I already done verification brother but there is no another option.

[ Edit ] Today i got it

**Reply by fury-05** [reply]
Wait for a day I will say.
